lldb/source/Symbol/UnwindPlan.cpp

```diff
@@ -153,6 +155,9 @@ void UnwindPlan::Row::RegisterLocation::Dump(Stream &s,
     if (m_type == atDWARFExpression)
       s.PutChar(']');
   } break;
+  case isConstant:
+    s.Printf("=%x", m_location.offset);
```

Comment:
Should the Printf be `("=0x%" PRIx64, m_location.constant_value)`?  If you do an unwind with one of these register locations in use, we should see this output when the unwind engine shows the unwind rules being used for this function.

Reply:
Ohh oops, you're totally right! Fixed!